server/sonar-server/src/main/resources/org/sonar/server/user/ws/current-example.json

```diff
@@ -12,5 +12,9 @@
   "groups": ["Jedi", "Rebel"],
   "permissions": {
     "global": ["profileadmin", "scan"]
+  },
+  "homepage": {
+    "type": "PROJECT",
```

Comment:
to remember: the value in parameters and in response should be the same (lowercase or uppercase)